invokeai/frontend/web/src/features/canvas/store/canvasSlice.ts

```diff
@@ -240,6 +240,16 @@ export const canvasSlice = createSlice({
         state.scaledBoundingBoxDimensions = scaledDimensions;
       }
     },
+    toggleBoundingBoxDimensions: (state) => {
+      const [currWidth, currHeight] = [
+        state.boundingBoxDimensions.width,
+        state.boundingBoxDimensions.height,
+      ];
+      state.boundingBoxDimensions = {
+        width: currHeight,
+        height: currWidth,
+      };
+    },
```

Comment:
I don't think `toggle` is the right word here, what we are doing is more like `flipBoundingBoxAxes`